src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -1348,44 +1291,48 @@ public partial class BinaryWriter : System.IDisposable, System.IAsyncDisposable
         public virtual void Write(uint value) { }
         [System.CLSCompliantAttribute(false)]
         public virtual void Write(ulong value) { }
-        public virtual void Write(System.ReadOnlySpan<byte> buffer) { }
-        public virtual void Write(System.ReadOnlySpan<char> chars) { }
         protected void Write7BitEncodedInt(int value) { }
     }
     public sealed partial class BufferedStream : System.IO.Stream
     {
-        public BufferedStream(Stream stream) { }
-        public BufferedStream(Stream stream, int bufferSize) { }
+        public BufferedStream(System.IO.Stream stream) { }
+        public BufferedStream(System.IO.Stream stream, int bufferSize) { }
+        public int BufferSize { get { throw null; } }
         public override bool CanRead { get { throw null; } }
         public override bool CanSeek { get { throw null; } }
         public override bool CanWrite { get { throw null; } }
         public override long Length { get { throw null; } }
         public override long Position { get { throw null; } set { } }
+        public System.IO.Stream UnderlyingStream { get { throw null; } }
         public override System.IAsyncResult BeginRead(byte[] buffer, int offset, int count, System.AsyncCallback callback, object state) { throw null; }
         public override System.IAsyncResult BeginWrite(byte[] buffer, int offset, int count, System.AsyncCallback callback, object state) { throw null; }
+        public override void CopyTo(System.IO.Stream destination, int bufferSize) { }
```

Comment:
New APIs that were missing before. Should they be exposed?

Same with https://github.com/dotnet/corefx/pull/35474/files#diff-f385e94efa80f66e8f1ee5d41032af30R1534
```C#
public override void Write(System.Text.StringBuilder value) { }
```